webpack/IopRecommendationDetails/IopRecommendationDetails.js

```diff
@@ -8,29 +8,19 @@ import { providerOptions } from '../common/ScalprumModule/ScalprumContext';
 const scope = 'advisor';
 const module = './RecommendationDetailsWrapped';
 
-const invScope = 'inventory';
-const invModule = './HybridInventoryTabs';
-
 const IopRecommendationDetails = props => {
   const urlParams = useRouteMatch('/foreman_rh_cloud/recommendations/:rule_id');
   // eslint-disable-next-line camelcase
   const ruleId = urlParams?.params?.rule_id;
   return (
-    <div className="rh-cloud-recommendation-details-cell">
+    <div className="iop-recommendation-details-scalprum">
```

Comment:
This may affect robotello tests. @jameerpathan111 .

Reply:
@ColeHiggins2